### DISCUSSION TITLE
starts_with for Bytes type

### DISCUSSION THREAD

**Discussion by dcz-self** [cozodb/cozo#132] [Ideas]
I'm storing hashes in the database as the Bytes type. The hashes are long enough that I don't always need to query the entire hash (see how git show $HASH does not need the entire hash, but just the beginning). Would it make sense to extend starts_with to Bytes? I'm ready to implement it.

**Comment by dcz-self** [comment]
Alternatively, a way to extract a slice from the bytes would also be helpful.

**Comment by zh217** [comment]
Good idea, I will do it now.
